library/psa_crypto_ecp.c

```diff
@@ -77,14 +77,14 @@ static int check_ecc_parameters(psa_ecc_family_t family, size_t *bits)
         case PSA_ECC_FAMILY_SECP_K1:
             switch (*bits) {
                 case 192:
-                case 224:
                 case 256:
                     return PSA_SUCCESS;
-                /* secp224k1 has 224-bit coordinates but 225-bit private keys.
-                 * This means that private keys are represented with 232 bits. */
+                /* secp224k1 is not and will not be supported in PSA (#3541).
+                 * Note: secp224k1 has 225-bit private keys which are rounded
+                 * up to 232 for their representation. */
+                case 224:
```

Comment:
This seems to contradict the comment, which has 225.

Reply:
225 is for the private key; I think public is still 224. Following "adjustments" I did for other families here I should change 232 to 225 for private keys, but since we're not supporting secp224k1 I didn't add this piece of code here. 
Btw I will fix the comment to be clearer.